packages/react/src/components/AnimatedHeader/components/AnimatedHeader/AnimatedHeader.tsx

```diff
@@ -258,7 +259,9 @@ const AnimatedHeader: React.FC<AnimatedHeaderProps> = ({
                   className={`${blockClass}__workspace`}
                   size="sm"
                   titleText="Label"
-                  label={`Open in: ${name}'s workspace` || 'Select a workspace'}
+                  label={
+                    `Open in: ${userName}'s workspace` || 'Select a workspace'
+                  }
```

Comment:
What do you think about exporting this label as prop as well? It will solve the internationalisation problems for it.

Reply:
No issue with doing this to fix the internationalization issue, although the designs recommend starting with a similar text format of `Open in: ${userName}'s workspace` depending on the language. Once a item is clicked from the dropdown it is removed essentially acting as just the placeholder.